packages/frontend-2/components/settings/workspaces/security/ConfirmJoinPolicyDialog.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
We should standardize some of these modals, most of them are just a title + text and buttons, so we can easily create a generic one. We do have one for unsaved changes but it has hardcoded copy, we could update that one. Not blocking for this one tho